test/NuGet.Core.Tests/NuGet.Commands.Test/MSBuildRestoreUtilityTests.cs

```diff
@@ -3398,6 +3398,27 @@ public void MSBuildRestoreUtility_AddPackageDownloads_InvalidVersionRange(string
             // Act & Assert
             var exception = Assert.Throws<ArgumentException>(() => MSBuildRestoreUtility.AddPackageDownloads(spec, msbuildItems));
         }
+        [Fact]
```

Comment:
Add exactly 1 empty line between methods.

```suggestion

        [Fact]
```